generator/transit_generator.cpp

```diff
@@ -1,16 +1,141 @@
 #include "generator/transit_generator.hpp"
 
+#include "generator/osm_id.hpp"
+
+#include "routing_common/transit_serdes.hpp"
+#include "routing_common/transit_types.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/file_name_utils.hpp"
+#include "coding/file_writer.hpp"
+
+#include "platform/platform.hpp"
+
+#include "base/checked_cast.hpp"
 #include "base/logging.hpp"
 #include "base/macros.hpp"
 
+#include "3party/jansson/src/jansson.h"
+
+using namespace platform;
+using namespace routing;
+using namespace routing::transit;
+using namespace std;
+
+namespace
+{
+/// \returns file name without extension(s) by a file path. That means it gets a file name by a full path and
```

Comment:
s/and and/and/

Reply:
I suggest you to shorten the comment to:
```c++
// Drops all extensions from the filename.
```

Reply:
Fixed

Reply:
> // Drops all extensions from the filename.

+1. Let's leave only: // Drops all extensions from the filename.